pkg/cmd/experimental/config/config.go

```diff
@@ -3,13 +3,29 @@ package config
 import (
 	"fmt"
 	"os"
+	"path"
 
-	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/config"
 	"github.com/spf13/cobra"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/config"
+
+	cmdconfig "github.com/openshift/origin/pkg/cmd/cli/config"
 )
 
 func NewCmdConfig(parentName, name string) *cobra.Command {
```

Comment:
Is there a reason for this to be under experimental, since it's also in `osc config` already?